test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -x
+
+git clone -b cleanUp  https://github.com/jstupak/aQGC.git
```

Comment:
once this is merged, folks can use master branch again.  So you should remove `-b cleanUp`